setup.md

```diff
@@ -92,33 +176,55 @@ may overflow your screen. Set the device scale under the **Window > Scale** menu
-4. Select your Development Team for the Flutter project.
-  1. Open the default Xcode workspace in your project under `ios/Runner.xcworkspace`. 
-  2. Select the `Runner` project in the left navigation panel.
-  3. In the Target Settings page, make sure your Development Team is selected under **General > Signing > Team**.
-5. Start your app by running `flutter run`.
+   ```
+   $ brew install ideviceinstaller ios-deploy
+   ```
+1. If you see the following error:
+    ```
+   You must brew link libtasn1 libplist libusb before ideviceinstaller can be installed
+    ```
+   run `brew link libtasn1 libplist libusb` to link the required libraries. 
+
+   If the command fails with an error related to directories that are not writeable, 
+   run `brew doctor` and follow the instructions for changing permissions on those directories.
+   Then run the `brew link ...` and `brew install...` commands again.
+   
+1. Follow the steps defined by Apple to configure Xcode with your Apple Developer account and enable deploying with provisioning profiles. To learn how to set up your profile, see [Apple's official documentation](https://developer.apple.com/library/ios/documentation/IDEs/Conceptual/AppDistributionGuide/MaintainingProfiles/MaintainingProfiles.html).
+1. Select your Development Team for the Flutter project.
+   1. Open the default Xcode workspace in your project under `ios/Runner.xcworkspace`. 
+   1. Select the `Runner` project in the left navigation panel.
+   1. In the Target Settings page, make sure your Development Team is selected under **General > Signing > Team**.
+1. Start your app by running `flutter run`.
 
 ## Android setup
 
 _This step is optional, and can be performed at any time before
 using Flutter to develop apps for Android._
 
+### Install Android Studio
+
 To develop Flutter apps for Android, you can use either a Mac or a Linux (64-bit) machine:
 
-1. Install [Android Studio](https://developer.android.com/sdk/index.html).
-2. From Android Studio, install the latest Android SDK and Android SDK Platform-Tools, as described
-in [Adding SDK Packages](https://developer.android.com/sdk/installing/adding-packages.html).
+1. Install [Android Studio](https://developer.android.com/studio/index.html).
+
+   Flutter requires the latest Android SDK and Android SDK Platform-Tools packages.
+In Android Studio, you can verify these are up-to-date, as described in [Update the IDE and SDK Tools](https://developer.android.com/studio/intro/update.html).
+
+1. Add the bundled JDK directory to your PATH variable. For example, on macOS:
+   ```
+   $ export PATH=/Users/obiwan/Applications/Android\ Studio.app/Contents/jre/jdk/Contents/Home/bin:$PATH
+   ```
+   or on Linux:
+   ```
+   $ export PATH=~/Android-Studio/jre/bin:$PATH
```

Comment:
Do we need to repeat the previous instructions to make this persist, or is being available once sufficient?